src/core/service.c

```diff
@@ -4769,6 +4769,22 @@ static void service_handoff_timestamp(
         unit_add_to_dbus_queue(u);
 }
 
+static void service_notify_pidref(Unit *u, PidRef *parent_pidref, PidRef *child_pidref) {
```

Comment:
Hmm, I would probably be more specific on what this is about, because it's really not obvious. Maybe, pidns_pidref or so?